scriptoocloud/src/main/webapp/WEB-INF/views/auth/projects.html

```diff
@@ -0,0 +1,27 @@
+<html layout:decorate="~{layouts/main}">
+  <head>
+    <title>Authority Matcher</title>
+  </head>
+  <body>
+	<div layout:fragment="content">
+	<div class="text-danger" th:text = "${deletionFailure}"></div>
+	<h2>Project List:</h2> 
+	  <ul class="list-group">
+		<li th:each="project : ${projects}" class="list-group-item clearfix">
+			<p>Name: <span th:text="${project.getName()}" th:remove="tag"></span> ID: <span th:text="${project.getId()}" th:remove="tag"></span> Description: <span th:text="${project.getDescription()}" th:remove="tag"></span> Date Created: <span th:text="${project.getCreationDate()}" th:remove="tag"></span> Last Modified: <span th:text="${project.getLastModifiedDate()}" th:remove="tag"></span></p>
+			<div class="pull-right text-right">
+			</div> 
+			
+			<form th:action="@{|/auth/projects/${project.getId()}/remove/|}" method="POST">
+		    	<button style="padding:0px" class="btn btn-link"><i class="fas fa-trash"></i> Delete Project</button>
+		    </form> 
```

Comment:
there should be a warning before the user actually deletes a project. Something that asks if the user is sure they want to delete the project since there will be a lot of content that would be deleted eventually.

Reply:
This functionality is apart of STC-11, will implement change there, and merge once that branch is done.